### ISSUE TITLE
Crash when placing an appliance on top of a conveyor

### ISSUE THREAD

**Issue by bloodyraoul** [Team-IO/taam#236]
```
java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: Cannot set property PropertyEnum{name=direction, clazz=class net.minecraft.util.EnumFacing, values=[south, west, north, east]} to down on block taam:productionline_appliance, it is not an allowed value
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_72]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_72]
	at net.minecraft.util.Util.func_181617_a(SourceFile:46) [h.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:668) [MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:387) [ld.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_72]
Caused by: java.lang.IllegalArgumentException: Cannot set property PropertyEnum{name=direction, clazz=class net.minecraft.util.EnumFacing, values=[south, west, north, east]} to down on block taam:productionline_appliance, it is not an allowed value
	at net.minecraftforge.common.property.ExtendedBlockState$ExtendedStateImplementation.func_177226_a(ExtendedBlockState.java:100) ~[ExtendedBlockState$ExtendedStateImplementation.class:?]
	at net.teamio.taam.content.conveyors.BlockProductionLineAppliance.func_176221_a(BlockProductionLineAppliance.java:73) ~[BlockProductionLineAppliance.class:?]
	at net.minecraftforge.common.ForgeHooks.canHarvestBlock(ForgeHooks.java:159) ~[ForgeHooks.class:?]
	at net.minecraftforge.common.ForgeHooks.blockStrength(ForgeHooks.java:198) ~[ForgeHooks.class:?]
	at net.minecraft.block.Block.func_180647_a(Block.java:532) ~[akf.class:?]
	at net.minecraft.block.state.BlockStateContainer$StateImplementation.func_185903_a(BlockStateContainer.java:395) ~[art$a.class:?]
	at net.minecraft.server.management.PlayerInteractionManager.func_180784_a(PlayerInteractionManager.java:205) ~[lv.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_147345_a(NetHandlerPlayServer.java:631) ~[me.class:?]
	at net.minecraft.network.play.client.CPacketPlayerDigging.func_148833_a(SourceFile:40) ~[ja.class:?]
	at net.minecraft.network.play.client.CPacketPlayerDigging.func_148833_a(SourceFile:10) ~[ja.class:?]
	at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) ~[fl$1.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_72]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_72]
	at net.minecraft.util.Util.func_181617_a(SourceFile:45) ~[h.class:?]
	... 5 more
```

**Comment by Xanderio** [comment]
can you please provide a better descripting what you did (e.g. screenshot)

**Comment by founderio** [comment]
Better yet: Which version (Mod + Minecraft) are you on?
I have an idea where this happened, but was not able to reproduce using latest build (1.1.0-beta11).

**Comment by Xanderio** [comment]
That is a gread idea Oliver +1

**Comment by bloodyraoul** [comment]
I play with the last version, the beta 11 with forge-1.10.2-12.18.1.2095...
I was just doing some tests in creative mode, putting some conveyor belt and appliance...

**Comment by founderio** [comment]
Okay, I will check once again, later. I tested with the 1.9 version locally.
I'll get back to you once I have checked.

**Comment by founderio** [comment]
Found it.
Happens when placing an appliance ON TOP of a conveyor. (Requires holding shift)

**Comment by founderio** [comment]
Incidentally my optimziation in placement code also seems to fix #207...

**Comment by founderio** [comment]
Fixed in beta 12 that is release just now. You can find it here: https://github.com/Team-IO/taam/releases Or later o CurseForge once they approved the file.
